### ISSUE TITLE
Simdoc plaftform dockerization

### ISSUE THREAD

**Issue by ChristopheMaldivi** [Orange-OpenSource/documentare-simdoc#33]
To ease deployment and run, we could package simdoc servers (UI & backends) in a docker image or several docker images (compose).

**Comment by JoelGardes** [comment]
Good idea for portability. What do you means by "compose" with several docker images,  I suppose that each docker image is completely independent ?

**Comment by ChristopheMaldivi** [comment]
see [here](https://docs.docker.com/compose/)

basically the idea behind this:
 - we keep a docker image for an atomic part of the platform, for instance : simdoc rest server is autonomous / user interface servers will also be an autonomous part
 - but to assemble these parts, docker compose will help us
 - it gives us the possibility to run and update these images in an independent way
 - these images are available in an internal or public docker registry and can be used by third party

Regarding this task, it could be a good entry point for @boisset :)

**Comment by JoelGardes** [comment]
I completely agree.

**Comment by ChristopheMaldivi** [comment]
Images are indeed independent. "Docker compose " tool can then be used to assemble several images to build and run a complete platform.


-------- Original message --------

**Comment by ChristopheMaldivi** [comment]
TODO:
 - deliver it to @JoelGardes , @boisset 
 - transfer the repo to OrangeOpensource@gitlab, @Christophe

Next: rebuild at each tag

**Comment by ChristopheMaldivi** [comment]
TODO:
 - check that shared directory contains debug data
 - add env variables for shared directory / memory settings
 - build docker image for tags and publish it on gitlab.com registry
